content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst

```diff
@@ -1,54 +1,209 @@
+   Check whether putaway rules work as intended and using the summary of internal product movements.
+   In the :guilabel:`Inventory` app, select :menuselection:`Reporting --> Product Moves` and enable
+   the :guilabel:`Internal` search filter on the :guilabel:`Filters` dropdown under the search bar
+   at the top of the page. In this view, confirm that products are moved to the :guilabel:`Store to`
+   location detailed in the putaway rule. More details in :ref:`this section
+   <inventory/putaway/wrong-location>`.
+
+Use putaway rules
+-----------------
+
+Once a putaway rule has been configured, the product it specifies is automatically routed to the
+:guilabel:`Store to` location upon arriving in the :guilabel:`When product arrives in` location.
+
+After configuring the putaway rules for apples and carrots, see the internal stock moves by
+first buying products from a vendor using a :ref:`purchase order<inventory/purchase/vendor bills>`
+(PO).
 
-.. image:: putaway/putaw2.png
+To create a :guilabel:`Request for Quotation` (RFQ), go to the :guilabel:`Purchase` app, and click
+:guilabel:`Create`. Then, add products to the :abbr:`RFQ (Request for Quotation)` by clicking
+:guilabel:`Add a line`, and selecting the desired product(s) from the drop-down menu in the
+:guilabel:`Product` column.
+
+Once ready, click :guilabel:`Confirm`. Doing so turns the :abbr:`RFQ (Request for Quotation)` into a
+:abbr:`PO (Purchase Order)`. Next, click :guilabel:`Send by Email` to send the :abbr:`PO (Purchase
+Order)` to the vendor if needed.
+
+.. image:: putaway/purchase-apples-and-carrots.png
    :align: center
+   :alt: Purchase order for apples and carrots
 
-Now, if I purchase apples and carrots to my supplier, they will be grouped in the same receipt but
-redirected to the right location automatically, thanks to putaway rules. This information is
-available from *Inventory Report*, under the reporting menu.
+Receive the incoming shipment by selecting :menuselection:`Receive products --> Validate`.
 
-.. image:: putaway/putaw3.png
+.. image:: putaway/receive-incoming-stock.png
    :align: center
+   :alt: Receive package of apples and carrots
+
+Verify stock moves
+==================
 
-.. image:: putaway/putaw4.png
+Confirm whether the putaway rule worked as intended and that the products have been moved to the
+correct location in the :guilabel:`Inventory Report`. View all products in stock in
+:menuselection:`Inventory --> Reporting --> Inventory Report`. By default, the products are
+grouped by location.
+
+Select the product, and the drop-down displays the locations the products currently are. In the
+image below, the locations shown are `WH/Stock/Fruits` and `WH/Stock/Vegetables`, which
+confirms that the putaway rules successfully.
+
+.. image:: putaway/destination-locations.png
    :align: center
+   :alt: Verify stock moves triggered by putaway rule
+
+For a more detailed report, click on a stock move line. On the report, the :guilabel:`Reference`
```

Comment:
I'm not sure what a "stock move line" is, and I can't find the "Reference". I think it would help to add an image here.